src/compiler/scala/tools/nsc/Reporting.scala

```diff
@@ -353,11 +364,13 @@ object Reporting {
 
     object JavaSource extends WarningCategory; add(JavaSource)
 
+    object Migration extends WarningCategory; add(Migration)
```

Comment:
Scala3Migration?

Reply:
Just update the name? Also, I haven't thought about behavior when going from 2.12 to -Xsource:3, but maybe there is nothing special to do.

Reply:
On 2.13 we only have migration warnings for going to 3.

I don't think we have any warnings on 2.12 about migrating to 3? 2.13 is required as an intermediate step.

I'd say `Migration` would be fine if there wasn't already `OtherMigration`. The Wconfs get a bit more noisy with `Scala3Migration` but that might be a good thing.